bindings/python/iree/runtime/setup.py.in

```diff
@@ -22,8 +22,10 @@ setup(
     description="IREE Python Runtime Components",
     long_description=README,
     long_description_content_type="text/markdown",
-    license="Apache-2.0",
+    license="Apache 2.0",
     classifiers=[
+        "Development Status :: 3 - Alpha",
+        "License :: OSI Approved :: Apache Software License",
```

Comment:
For reference:

https://github.com/google/jax/blob/main/setup.py
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py